### PR TITLE
[app-review] updated mobile slides

### PR DESCRIPTION
**Pull request by chrisdugne**
**Detailed purpose of the PR**

updated mobile app-review:

- added `autoplay` for jwplayer and youtube
- `medias` are now part of the scrolling area
- question headers now handles `html`
- `skin` is not required on mobile options

additional fix:

- fixed a rendering test on package app-review

### CODE REVIEW

**Comment by chrisdugne** [comment]
[[OR] Recette PR-829](https://trello.com/c/Bxcd1BZo/2990-or-recette-pr-829)

**Review by jomaora — APPROVED** [review]

File: packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx [hunk 1]
```diff
@@ -178,22 +182,27 @@ const Question = (props: QuestionProps) => {
...
+  const hasVideoOrImage =
+    answerUI.media?.type && [TYPE_VIDEO, TYPE_IMAGE].includes(answerUI.media.type);
+
   return (
     <>
       <View style={style.questionHeading}>
-        <Text style={style.questionOrigin}>{questionOrigin}</Text>
-        <Text style={style.questionText}>{questionText}</Text>
-        <Text style={style.questionHelp}>{get('help', answerUI)}</Text>
+        <Html style={style.questionOrigin}>{questionOrigin}</Html>
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx:191]: any reason for moving from Text to HTML ?

**Review by scalabw — APPROVED** [review]

File: packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx (hunk 1 above)
Comment (chrisdugne) [reply packages/@coorpacademy-components/src/organism/review-slide/index.native.tsx:191]: these portions of question may be formated as html when defined on cockpit

**Comment by chrisdugne** [comment]
[[OR] Recette PR-829](https://trello.com/c/Bxcd1BZo/2990-or-recette-pr-829)

**Pull request merged**